### PR TITLE
Refactor: correction in installer commands.

### PR DESCRIPTION
**Pull request by vivek-webkul**
1) Not able to see the highlighted text:
Screenshot from 2024-01-06 11-52-52

Should be like this:
Screenshot from 2024-01-06 12-08-43

2) These options must be required of any default value must be selected:
Screenshot from 2024-01-06 11-57-55

Fixed.

### CODE REVIEW

File: packages/Webkul/Installer/src/Console/Commands/Installer.php [hunk 1]
```diff
@@ -182,43 +182,49 @@ protected function askForApplicationDetails()
...
+            }
+        } while (empty($allowedCurrencies));
+
         $defaultLocale = $this->updateEnvChoice(
             'APP_LOCALE',
-            'Please select the <bg=green>default application locale</>',
-            $this->locales
+            'Please select the <bg=black;fg=white;options=bold>default application locale</>',
+            $allowedLocales
         );
```
Comment (devansh-webkul) [inline packages/Webkul/Installer/src/Console/Commands/Installer.php:215]: Sir,

This is admin panel locale, not the default locale.

File: packages/Webkul/Installer/src/Console/Commands/Installer.php [hunk 2]
```diff
@@ -182,43 +182,49 @@ protected function askForApplicationDetails()
...
 
         $defaultCurrency = $this->updateEnvChoice(
             'APP_CURRENCY',
-            'Please select the <bg=green>default currency</>',
-            $this->currencies
+            'Please select the <bg=black;fg=white;options=bold>default currency</>',
+            $allowedCurrencies
         );
```
Comment (devansh-webkul) [inline packages/Webkul/Installer/src/Console/Commands/Installer.php:221]: Same here,

This is the admin panel currency which is set in the .env file, not in the channel default currency.

File: packages/Webkul/Installer/src/Console/Commands/Installer.php (hunk 1 above)
Comment (vivek-webkul) [reply packages/Webkul/Installer/src/Console/Commands/Installer.php]: Reverted in the latest commit.

File: packages/Webkul/Installer/src/Console/Commands/Installer.php (hunk 2 above)
Comment (vivek-webkul) [reply packages/Webkul/Installer/src/Console/Commands/Installer.php]: Reverted in the latest commit.

**Pull request closed without merging**